### PR TITLE
Change mempool defaults to reduce the chance of double spends.

### PR DESCRIPTION
**Pull request by ghost**
This should significantly reduce the chance of transaction eviction which will reduce the chance of a double spend.

### CODE REVIEW

File: src/main.h [hunk 1]
```diff
@@ -57,7 +57,7 @@ static const unsigned int DEFAULT_DESCENDANT_LIMIT = 25;
 /** Default for -limitdescendantsize, maximum kilobytes of in-mempool descendants */
 static const unsigned int DEFAULT_DESCENDANT_SIZE_LIMIT = 101;
 /** Default for -mempoolexpiry, expiration time for mempool transactions in hours */
-static const unsigned int DEFAULT_MEMPOOL_EXPIRY = 72;
+static const unsigned int DEFAULT_MEMPOOL_EXPIRY = 720;
```
Comment (orangeman) [inline src/main.h:60]: NACK :x:  that's too risky imho

**Double spending is a serious problem.**
Any better ideas how we could solve this?

Why have https://github.com/bitcoinclassic/bitcoinclassic/pull/16 and https://github.com/bitcoinclassic/bitcoinclassic/pull/15 been shut down without democrativc voting?  
Why was the other pullrequest deleted that discussed the removal of RBF?

**Comment by zander** [comment]
Please explain why you think this is better, and please give some testing output you gathered that led you to these magical values :)

I'm sceptical that this will work on memory constrained nodes, what do you think about that?

**Comment by orangeman** [comment]
@zander You seem to have missed the previous discussion. Please read https://github.com/bitcoinclassic/bitcoinclassic/pull/4 https://github.com/bitcoinclassic/bitcoinclassic/pull/16 https://github.com/bitcoinclassic/bitcoinclassic/pull/17 https://github.com/bitcoinclassic/bitcoinclassic/pull/22 and https://github.com/bitcoinclassic/bitcoinclassic/pull/14

**Comment by dgenr8** [comment]
NACK.  Proposes changes with serious resource requirements and no consideration of the cons.

**Comment by zander** [comment]
@orangeman I read all those, nowhere is any technical description given like I asked. Just changing numbers with no explanation why they ended up what they became is never going to get used. So thats why I asked.

**Comment by jtoomim** [comment]
Merge window is open only for 2 MB HF stuff.

An exception for this PR is not justified. A [vote](https://bitcoinclassic.consider.it/merge-19-change-mempool-defaults-to-reduce-the-chance-of-double-spends?results=true) has been started on consider.it. It does not appear to be a favorable change. 

I am closing this. It may be reopened after the fork.

**Pull request closed without merging**